website/src/app/api.service.ts

```diff
@@ -0,0 +1,73 @@
+import { Injectable } from '@angular/core';
+import { HttpModule, Http } from '@angular/http';
+import 'rxjs/Rx';
+import { Observable } from 'rxjs/Observable';
+import { User } from './user'
+import { environment } from 'src/environments/environment';
+
+const API_URL = environment.apiUrl;
```

Comment:
This should be more like a private property of the class, also not really needed since environment.apiUrl is not bad.